src/components/MovieDetailModal/render.ts

```diff
@@ -0,0 +1,130 @@
+import { BASE_IMAGE_URL } from '../../constants/api/api';
+import { Genre } from '../../types/tmdb';
+import createElement from '../../utils/createElement';
+import NoImage from '../ui/NoImage';
+import RatingStars from '../RatingStars/RatingStarts';
+import formatToDecimalPlaces from '../../utils/formatToDecimalPlaces';
+import { THUMBNAIL_SIZE } from '../../constants/ui';
+
+type MovieDetailProps = {
+  id: number;
+  title: string;
+  poster_path: string;
+  genres: Genre[];
+  overview: string;
+  vote_average: number;
+};
+
+const renderHandler = ({ id, title, poster_path, genres, overview, vote_average }: MovieDetailProps) => {
+  const { movieDetail, movieDetailContainer } = createMovieDetailContainer();
+  const genresText = genres.map((genre) => genre.name).join(', ');
```

Comment:
```suggestion
  const genresText = genres.map(({name) => name).join(', ');
```
이런 것은 또 구조분해할당하면 좋을 것 같아요!

Reply:
깃 커밋 주소 : 3eb643985511554fe8eb566ae6f44ede51fd1b0b

변수 사용할 때는 구조 분해 할당 하지 않고, 변수를 사용하지 않을 때는 구조 분해 할당을 해놨네요.. 수정했습니다!